otcextensions/osclient/cce/v2/cluster.py

```diff
@@ -181,47 +204,35 @@ def get_parser(self, prog_name):
             type=int,
             help=_('Interval for checking status')
         )
+        parser.add_argument(
+            '--wait-timeout',
+            type=int,
+            help=_('Wait timeout')
+        )
         return parser
 
     def take_action(self, parsed_args):
 
         attrs = {}
-
-        attrs['name'] = parsed_args.name
-        spec = {}
-        spec['type'] = parsed_args.type
-        spec['flavor'] = parsed_args.flavor
-        if parsed_args.version:
-            spec['version'] = parsed_args.version
-        if parsed_args.description:
-            spec['description'] = parsed_args.description
-        host_net = {
-            'router_id': parsed_args.router_id,
-            'network_id': parsed_args.network_id
-        }
-        if spec['type'] == 'BareMetal':
-            host_net['highwaySubnet'] = parsed_args.highway_subnet
-        spec['hostNetwork'] = host_net
-        container_net = {
-            'mode': parsed_args.container_net_mode
-        }
-        if parsed_args.container_net_cidr:
-            container_net['cidr'] = parsed_args.container_net_cidr
-        spec['containerNetwork'] = container_net
-
-        attrs['spec'] = spec
-
-        client = self.app.client_manager.cce
-
-        obj = client.create_cluster(**attrs)
-
-        if obj.job_id and parsed_args.wait:
-            wait_args = {}
-            if parsed_args.wait_interval:
-                wait_args['interval'] = parsed_args.wait_interval
-
-            client.wait_for_job(obj.job_id, **wait_args)
-            obj = client.get_cluster(obj.id)
+        for attr in [
+            'name', 'type', 'flavor', 'router', 'network',
+            'container_network_mode', 'annotations', 'authentication_proxy_ca',
+            'authentication_mode', 'container_network_cide',
+            'cpu_manager_policy', 'dss_master_volumes', 'description',
+            'external_ip', 'fix_pool_mask', 'labels', 'service_ip_range',
```

Comment:
what do you think about renaming `labels` to `k8slabels`?

Reply:
it's all belonging to k8. Not sure it make any sense. Generally here all names are pretty much how they are called in API with some normalizations.